artifacts/scripts/publish_template.sh

```diff
@@ -89,6 +89,7 @@ EXTRA_ARGS=()
 PUSH_SCRIPT=../push-tags-${REPO}-${DST_BRANCH}.sh
 echo "#!/bin/bash" > ${PUSH_SCRIPT}
 chmod +x ${PUSH_SCRIPT}
-/sync-tags --upstream-remote upstream-kube --upstream-branch "${SRC_BRANCH}" \
+/sync-tags --repo-org kubernetes --repo-name kubernetes \
```

Comment:
we need the right values here, don't we?

Reply:
fixed.